spec/service/zendesk_ticket_service_spec.rb

```diff
@@ -39,33 +42,33 @@
 
       expect(request
         .with(headers: {
-          "Authorization" => "Basic emVuZGVza191c2VyQHRlc3QuZXhhbXBsZS90b2tlbjpzdXBlcnNlY3JldA==",
-        })).to have_been_made
+                "Authorization" => "Basic emVuZGVza191c2VyQHRlc3QuZXhhbXBsZS90b2tlbjpzdXBlcnNlY3JldA=="
+              })).to have_been_made
     end
 
     it "raises an exception if the request fails" do
       stub_request(:post, "https://test.zendesk.com/api/v2/tickets.json")
         .to_return_json(status: 401, body: { "error" => "Couldn't authenticate you" })
 
-      expect {
+      expect do
         described_class.create!(comment: { body: "Test" })
-      }.to raise_error(/Creating Zendesk ticket failed/)
+      end.to raise_error(/Creating Zendesk ticket failed/)
```

Comment:
This is a breaking change with rubocop-govuk:

https://github.com/alphagov/rubocop-govuk/blob/7dc2cb6bafa0f605cf86fcf98f1e08505d11f17b/config/style.yml#L10-L14